<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_metodo_newton</class>
 <widget class="QDialog" name="ventana_metodo_newton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metodo Newton-Raphson</string>
  </property>
  <widget class="QPushButton" name="boton_newton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Método de newton</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="mitabla">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>404</width>
     <height>39</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Valor de x,y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valores">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>80</y>
     <width>148</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ecuación 1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="expresion1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>404</width>
     <height>39</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ecuación 2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="expresion2">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="boton_puntofijo">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Punto fijo</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>404</width>
     <height>39</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cantidad iteraciones:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iteraciones">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
